external/hbase/src/main/scala/org/apache/gearpump/external/hbase/HBaseSink.scala

```diff
@@ -148,11 +158,24 @@ object HBaseSink {
     val userName = userConfig.getString(HBASE_USER)
     if (userName.isEmpty) {
       ConnectionFactory.createConnection(configuration)
+
     } else {
       val user = UserProvider.instantiate(configuration)
         .create(UserGroupInformation.createRemoteUser(userName.get))
       ConnectionFactory.createConnection(configuration, user)
     }
 
+
+  }
+  def getConn(connection: Connection, isTest: Boolean, userConfig: UserConfig,
```

Comment:
We don't need `isTest` here since this method is not supposed to be called in test